packages/datastore-storage-adapter/__tests__/ExpoSQLiteDatabase.test.ts

```diff
@@ -0,0 +1,287 @@
+import ExpoSQLiteDatabase from '../src/ExpoSQLiteAdapter/ExpoSQLiteDatabase';
+import { openDatabase } from 'expo-sqlite';
+import { ParameterizedStatement } from '../src/common/types';
+import { HelperStack } from 'typedoc/dist/lib/output/utils/resources/helpers';
+
+// mock out the underlying implementation and only keep openDatabase().
+jest.mock('expo-sqlite', () => {
+	return {
+		__esModule: true,
+		openDatabase: jest.fn(() => mockDatabase),
+	};
+});
+
+// mock out the underlying implementation
+jest.mock('expo-file-system', () => 'File System');
+
+const mockDatabase = {
+	readTransaction: jest.fn(),
+	transaction: jest.fn(),
+};
+
+const mockTransaction = {
+	executeSql: jest.fn(),
+};
+
+const mockResultSet = jest.fn();
+
+const MOCK_SQL_ERROR = {
+	code: 400,
+	message: 'Error performing the transaction',
+};
+
+describe('ExpoSQLiteAdapter', () => {
+	const ExpoSQLiteAdapterInstance = new ExpoSQLiteDatabase();
+	ExpoSQLiteAdapterInstance.init();
```

Comment:
The `init()` function should also be tested

Reply:
Yes. I have added tests for `init()` and all other methods.